src/common/KernelBase.cpp

```diff
@@ -575,16 +575,30 @@ void KernelBase::doOnceCaliMetaBegin(VariantID vid, size_t tune_idx)
 {
   if(doCaliMetaOnce[vid].at(tune_idx)) {
     // attributes are class variables initialized in ctor
-    cali_set_double(ProblemSize_attr,(double)getActualProblemSize());
```

Comment:
Should these use whatever int type caliper uses instead of the RAJAPerf Index_type?

Reply:
`cali_set_int` is 32-bit int which will overflow the bytes metrics for larger problem sizes. Ideally we would use a `cali_set_int64` function (doesn't exist), but `cali_set` is capable of storing the value of `Index_type`.